<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>perspectiveTransformationWindow</class>
 <widget class="QMainWindow" name="perspectiveTransformationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>透视变换</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>701</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;微软雅黑&quot;;
font: 18pt &quot;Adobe Devanagari&quot;;
font: 14pt &quot;Adobe Devanagari&quot;;</string>
    </property>
    <property name="text">
     <string>1、由于照片的差异，需要手动调整一些参数，帮助系统更好的运行。
2、强度梯度小于minThresh的必定不是边缘
3、强度梯度大于maxThresh的必定是边缘
4、dilate会连接不连续部分
5、erode会丢弃边缘部分的像素，可能会造成不连续
6、请随机选择一批图片中的一张
7、点击Esc退出settings界面
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPerspectiveTransformationAddress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>631</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonpersPectiveTransformationChoice">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>290</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(177, 177, 177);</string>
    </property>
    <property name="text">
     <string>选择图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPerspectiveTransformationImage">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>240</width>
      <height>320</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"># labelBasicParameterImage{
rgb(255, 255, 127)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartAdjust">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始调整</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>处理结果：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOriginalperspectiveTransformationImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>240</width>
      <height>320</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"># labelBasicParameterImage{
rgb(255, 255, 127)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>原图：</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBarPerspectiveTransformation"/>
 </widget>
 <resources/>
 <connections/>
</ui>
